posthog/queries/test/test_retention.py

```diff
@@ -422,7 +422,7 @@ def test_retention_people_first_time(self):
             )
 
             self.assertEqual(len(result), 1)
-            self.assertEqual(result[0]["id"], p3.pk)
+            self.assertTrue(result[0]["id"] == p3.pk or result[0]["id"], p3.uuid)
```

Comment:
Typo here with the comma. This is always truthy?

Maybe extract a `get_actor_id` function in tests instead?

Reply:
leaving for now. once we purge postgres stuff these tests will be tightened up